source/usage-examples/updateMany.txt

```diff
@@ -0,0 +1,113 @@
+===========================
+Update a Multiple Documents
+===========================
+
+.. default-domain:: mongodb
+
+You can update multiple documents using the ``updateMany()`` method on
+a ``MongoCollection`` object. The method accepts a **filter** that matches the
+document you want to update and an **update** statement that instructs the
+driver to change the matching document. The ``updateMany()`` method only
+updates all the documents in the collection that match the filter.
```

Comment:
personally I feel that the "only...all" construction here is a little hard to parse. Do you think we could remove "only", or is there a good reason for it to remain here?

Reply:
Whoops, good catch. Will fix.